openshift4_aws/roles/deploy_web_terminal/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
We may want to update this file if you feel it would be useful... otherwise just delete it since it doesn't have info specific to the role.